### PR TITLE
E77 easy solder board: add building notes

### PR DESCRIPTION
**Pull request by olliw42**
an attempt at providing some build help

### CODE REVIEW

File: docs/E77_EASYSOLDER.md [hunk 1]
```diff
@@ -66,4 +66,31 @@ Notes:
...
+- C4,C6: 100n, RM2
+- IC1: LD1117AV33, TO220, low drop fixed voltage regulator 3.3 V, 1 A 
+- IC2: Ebyte E77-900M22S
+- IC3: Ebyte E22-900M22S
+- LED1:	red LED 1.8 mm
+- LED2:	green LED 1.8 mm
+- SW1: button 6 x 3.5 mm
+- D1: BAT42, Schottky diode
+
+For convenience, a Mouser list is available: https://eu.mouser.com/ProjectManager/ProjectDetail.aspx?AccessID=b60c19cf3a&_gl=1*nmaj37*_ga*OTEyNjU5NzcwLjE2NzU2MzU2MjQ.*_ga_15W4STQT4T*MTY5NjI2NzIzMS4xMS4wLjE2OTYyNjcyMzIuNTkuMC4w.
```
Comment (jlpoltrack) [inline docs/E77_EASYSOLDER.md:89]: Could use 'here' descriptor so that user doesn't have to see the entire link.

File: docs/E77_EASYSOLDER.md [hunk 2]
```diff
@@ -66,4 +66,31 @@ Notes:
...
 - From the Read Out Protection section, change to AA, select Apply
-    - This will erase the current firmware
\ No newline at end of file
+    - This will erase the current firmware
+
+## Building ##
+
+As mentioned in the introduction, with the exception of the E77 and E22 modules the board uses only through-hole parts, which makes soldering as simple as possible. Moreover, standard parts are mostly used, which should help with sourcing. Only the two 1.8 mm LEDs are somewhat special and less common, but can be substituted with standard 3 mm LEDs if necessary. 
+
+The electronic scheme and board design files, as well as Gerber files for ordering PCBs, can be found here: https://github.com/olliw42/mLRS-hardware/tree/master/olliw-stm32-based/rx-tx-E77-E22-dual-easysolder.
```
Comment (jlpoltrack) [inline docs/E77_EASYSOLDER.md:75]: Could use 'here' descriptor so that user doesn't have to see the entire link.

File: docs/E77_EASYSOLDER.md [hunk 3]
```diff
@@ -66,4 +66,31 @@ Notes:
...
+- IC2: Ebyte E77-900M22S
+- IC3: Ebyte E22-900M22S
+- LED1:	red LED 1.8 mm
+- LED2:	green LED 1.8 mm
+- SW1: button 6 x 3.5 mm
+- D1: BAT42, Schottky diode
+
+For convenience, a Mouser list is available: https://eu.mouser.com/ProjectManager/ProjectDetail.aspx?AccessID=b60c19cf3a&_gl=1*nmaj37*_ga*OTEyNjU5NzcwLjE2NzU2MzU2MjQ.*_ga_15W4STQT4T*MTY5NjI2NzIzMS4xMS4wLjE2OTYyNjcyMzIuNTkuMC4w.
+
+Notes:
```
Comment (jlpoltrack) [inline docs/E77_EASYSOLDER.md:91]: Small suggestion, use asterisk to connect parts in BOM with notes (similar to CRSF sensors page): https://github.com/olliw42/mLRS-docu/blob/master/docs/CRSF_SENSORS.md

For example, add * to C5, C6, IC3 and then use * in the notes to denote that these are needed for diversity.

**Comment by jlpoltrack** [comment]
Looks good to me, just a few small suggestions.

File: docs/E77_EASYSOLDER.md (hunk 1 above)
Comment (olliw42) [reply docs/E77_EASYSOLDER.md]: done

File: docs/E77_EASYSOLDER.md (hunk 2 above)
Comment (olliw42) [reply docs/E77_EASYSOLDER.md]: done

File: docs/E77_EASYSOLDER.md (hunk 3 above)
Comment (olliw42) [reply docs/E77_EASYSOLDER.md]: pl check if it's how you envisioned it

**Comment by olliw42** [comment]
many ths for the comments. I hope to have accomodated them. Also did some few more changes, especially to clarify D1 better.

**Comment by jlpoltrack** [comment]
Looks good to me - will merge today if you're all set.

**Comment by olliw42** [comment]
I'm all set :)

**Pull request merged**